Website/Packages/datax-pipeline/src/modules/flowDefinition/components/flowDefinitionPanel.jsx

```diff
@@ -37,8 +32,17 @@ import {
     IconButtonStyles,
     VerticalTabs,
     VerticalTabItem,
-    getApiErrorMessage
+    getApiErrorMessage,
```

Comment:
nit: remove ending comma

Reply:
Updated. (I would have expected this to throw a compilation error)

Reply:
this doesn't throw an error in C# either. Its pretty much the same behavior across several languages such as C++, Java, etc.